tests/datasets/test_carnatic_rhythm.py

```diff
@@ -1,7 +1,15 @@
 import os
 import numpy as np
 from mirdata import annotations
-from mirdata.datasets import compmusic_carnatic_rhythm
+
+try:
+    from mirdata.datasets import compmusic_carnatic_rhythm
+except ImportError:
+    raise ImportError(
+        "In order to use CompMusic Carnatic Music Rhythm you must have openpyxl installed. "
```

Comment:
I would propose this just because the exception is a bit more general and it can fail in other ways rather than missing dependencies. 
"An error occured when importing this dataset. Most likely this is due to a dependency not being installed, in this case openpyxl."

Reply:
sure, I'll make the changes.